### PR TITLE
Update to index page and sidebar

### PR DESCRIPTION
**Pull request by rbsec**
The current text is....not great. This removes a lot of the old and legacy fluff, and gives us a much cleaner page.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -15,47 +14,38 @@ type: documentation
...
+## Project Leaders
 
-> Requests from OPC/OASVS are flagged with a special label in the GitHub repository issues list in order to identify them and set them as a top level priority.
+- [Jim Manico](mailto:jim.manico@owasp.org)
 
 ## Core Team
 
-- [Jakub Maćkowski](https://github.com/mackowski)
-- [Robin Bailey](https://github.com/rbsec/)
+- [Elie Saad](mailto:eliesaad7@gmail.com)
```
Comment (mackowski) [inline index.md]: Move Elie to **Project Leaders** section

**Review by mackowski — COMMENTED** [review]
I am fine with the changes proposed.

File: index.md (hunk 1 above)
Comment (rbsec) [reply index.md]: Fixed

**Review by ThunderSon — APPROVED** [review]
Thanks!

**Pull request merged**